.github/workflows/build_python.yml

```diff
@@ -0,0 +1,30 @@
+name: build_python
+# build for python, skip scala tests and R build
+on:
+  push:
+    branches:
+      - "python/*"
+  pull_request:
```

Comment:
The PR should run all builds, not only python.... Right?

Reply:
PR will run the build_everything step - this includes all the sub builds, so we don't need to explicitly run against PR. The logic I was working with was;
- coder works on a specific language branch.
- pushing against that branch runs the build relevant to only that code. The idea here is that when adding scala functionality the only tests that are relevant are scala. Once the scala code is complete and working and a PR is made, the full build executes to identify any issues introduced into other line.

Reply:
I've updated the build_main to run on all non code specific branches, and PRs as well. If it passes review, good for merge I think

Reply:
Great!